.github/workflows/generate-typedoc.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate TypeDoc
+
+on:
+  workflow_dispatch:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+
+      - name: Set node version to 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Generate TypeDoc
+        run: |
+          npm ci
+          npm run typedoc
+
+      - name: Commit files
+        if: "github.repository == 'pioardi/poolifier' && github.ref == 'refs/heads/master'"
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Does the number have a special meaning? Or it's just random?

Reply:
It's coming from the examples: https://github.com/ad-m/github-push-action#example-workflow-file